docs/pages/versions/unversioned/sdk/battery.mdx

```diff
@@ -23,44 +23,41 @@ import { SnackInline} from '~/ui/components/Snippet';
 <SnackInline label='Basic Battery Usage' dependencies={['expo-battery']}>
 
 ```jsx
-import * as React from 'react';
-import * as Battery from 'expo-battery';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useEffect, useState } from "react";
```

Comment:
```suggestion
import { useEffect, useState } from "react";
```

I don't think, for this example, `React` needs to be imported explicitly since it is not used in the code below.